tests/bootstrap_project/test/project_test.dart

```diff
@@ -0,0 +1,432 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' as path;
+
+void main() async {
+  Directory.current = path.join(Directory.current.path, '..', '..');
+  final rootPath = Directory.current.path;
+  final cliPath = path.join(rootPath, 'tools', 'serverpod_cli');
+  final tempPath = path.join(rootPath, 'temp');
+
+  final timeout = Timeout(Duration(minutes: 5));
+
+  setUpAll(() async {
+    await Process.run(
+      'dart',
+      ['pub', 'global', 'activate', '-s', 'path', '.'],
+      workingDirectory: cliPath,
+    );
+
+    await Process.run('mkdir', ['temp'], workingDirectory: rootPath);
+    Directory.current = tempPath;
+  });
+
+  group('Given a clean state', () {
+    final (projectName, commandRoot) = createRandomProjectName(tempPath);
+
+    late Process createProcess;
+
+    tearDown(() async {
+      createProcess.kill();
+
+      await Process.run(
+        'docker',
+        ['compose', 'down', '-v'],
+        workingDirectory: commandRoot,
+      );
+    });
+
+    test(
+        'when creating a new project then the project is created successfully and can be booted',
+        () async {
+      createProcess = await Process.start(
+        'serverpod',
+        ['create', projectName, '-v'],
+        workingDirectory: tempPath,
+        environment: {
+          'SERVERPOD_HOME': rootPath,
+        },
+      );
+
+      createProcess.stdout.transform(Utf8Decoder()).listen(print);
+      createProcess.stderr.transform(Utf8Decoder()).listen(print);
+
+      var createProjectExitCode = await createProcess.exitCode;
+      expect(createProjectExitCode, 0);
+
+      final docker = await Process.start(
+        'docker',
+        ['compose', 'up', '--build', '--detach'],
```

Comment:
Question - When we run this with `--detach`, isn't this a high risk of creating a Zoombie process? Wouldn't it be better to run this command asynchronously and close it once the test is finished?

Reply:
Since we retry until the database is up when we start the server in maintenance mode this should be OK as well.

Reply:
The reason I did this was to ensure the container has started before we run our command. But I think the risk of a zombie is just as big without using --detach as we would have to kill the process manually or it will continue to run in the background. 

Now the tearDown will kill the containers.